core/src/cluster_slots_service.rs

```diff
@@ -81,10 +81,8 @@ impl ClusterSlotsService {
     ) {
         let mut cluster_slots_service_timing = ClusterSlotsServiceTiming::default();
         let mut last_stats = Instant::now();
-        loop {
-            if exit.load(Ordering::Relaxed) {
-                break;
-            }
+        let mut epoch_specs = EpochSpecs::from(bank_forks.clone());
+        while !exit.load(Ordering::Relaxed) {
```

Comment:
in the future, please avoid superfluous refactors like this for changes that are intended for backport. they can go into master as a followup

Reply:
Sure thing!